app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -1,3 +1,32 @@
+import { buildSelectOptions } from "utils";
+import * as yup from "yup";
+
+export const CONTACTS_FORM_INITIAL_FORM_VALUES = {
+  firstName: "",
+  lastName: "",
+  role: null,
+  email: "",
+};
+
+export const ROLES = buildSelectOptions(["User", "Admin", "developer"]);
+
+export const CONTACTS_FORM_VALIDATION_SCHEMA = yup.object().shape({
+  firstName: yup.string().required("First name is required"),
+  lastName: yup.string().required("Last name is required"),
+  role: yup
+    .object()
+    .nullable()
+    .shape({
+      label: yup.string().oneOf(ROLES.map(tag => tag.label)),
+      value: yup.string().oneOf(ROLES.map(tag => tag.value)),
+    })
+    .required("Role is required"),
+  email: yup
+    .string()
+    .required("Email is required")
+    .email("Email format incorrect"),
```

Comment:
So when we point out a validation error with red border, it's already implied that the value we have provided is `incorrect` and thus the error. So it doesn't help the consumer if we again point out the obvious fact in the error message too. Rather here we should construct the error message with a suggestive/solution tone.

Example:
```suggestion
.email("Email should have a valid format"),
```